### PR TITLE
Fix false intercalary first month

### PR DESCRIPTION
**Pull request by n-riesco**
This PR includes PR #8

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -282,4 +282,62 @@ describe('Chinese calendar', function() {
...
+    });
+
+    it('should format dates in the first month correctly', function() {
+        for(var year = 1888; year < 2112; year++) {
+            var newYearString = year + "/01/01";
+            var newYear = chineseCalendar.parseDate(null,newYearString); 
+            expect(newYear.formatDate()).toBe(newYearString);
+
+            var intercalaryMonth = chineseCalendar.intercalaryMonth(year);
+            if (intercalaryMonth === 1) {
```
Comment (alexcjohnson) [inline test/test.js:336]: does this ever happen?

**Comment by alexcjohnson** [comment]
@n-riesco thanks for the quick fix + tests!

**Comment by alexcjohnson** [comment]
btw - I had missed #8 entirely until I came to look for this PR - if there are any more just @ mention me in the PR so I'm sure to see it!

File: test/test.js (hunk 1 above)
Comment (n-riesco) [reply test/test.js:336]: Not in the range we currently cover (1888-2111), but [Aslaksen's paper (Table 23)](http://www.math.nus.edu.sg/aslaksen/calendar/cal.pdf) quotes 6 occurrences between 1645 and 2644.

---

Note to my future me: there are no intercalary 12th months between 1645 and 2644.

**Pull request merged**